tools/missing-test-detector/testdata/covered_resource_test.go

```diff
@@ -5,10 +5,11 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/acctest"
```

Comment:
I am so familiar how the tool `missing-test-detector` works. Does it matter to import the TPG or TPGB provider?

Reply:
No, this is just a test file that never actually gets compiled.